packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+# 48.13.0 - April 30, 2020
+
+## Fixed
+
+#### salesforcedx-vscode-lwc
+
+- Add check for local dev server start failure ([PR #2128](https://github.com/forcedotcom/salesforcedx-vscode/pull/2128))
+
+## Added
+
+#### docs
+
+- Beta for source retrieve on Apex, Visualforce, Aura and LWC ([PR #2102](https://github.com/forcedotcom/salesforcedx-vscode/pull/2102))
```

Comment:
You can remove this and just add a link to the Perf Enhancement docs in the line for this work under the `salesforcedx-vscode-core` section

Reply:
Hope I got the link to public Perf Enhancement docs link right. Please confirm.